wandb/sdk/data_types/video.py

```diff
@@ -100,6 +101,17 @@ def __init__(
                 "wandb.Video accepts {} formats".format(", ".join(Video.EXTS))
             )
 
+        # TODO: Throw an error if fps is provided when passing raw bytes, or file path.
+        # For now display warning if fps is provided when passing raw bytes, or file path.
+        if isinstance(data_or_path, (BytesIO, str)) and fps:
+            if fps:
+                deprecate.deprecate(
```

Comment:
why deprecate? are we planning to deprecate this attribute?

Reply:
I wasn't sure of the appropriate way to make this change.
@timoffex and I discussed adding a warning for some time before throwing an error when providing the fps argument and a path to the file or bytes.

Is the best way rather to do `logger.warn` instead?

Reply:
we use `wandb.termerror` for errors and `wandb.termwarn` for warning.
if you read the contributing doc (https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md#deprecating-features) you could see that deprecate.deprecate feature is used for when we want collect telemetry before deprecating a feature, which is not the case here as far as i understand

Reply:
you could add telemetry just to see how many users actually use this argument in the way where they expect it to re-encode their videos